deploy/chart/templates/plugin.yaml

```diff
@@ -193,6 +193,9 @@ spec:
 {{- if $nodePool.csi.oss.enabled }}
             - mountPath: /host/var/run/ossfs
               name: ossfs-metrics-dir
+            - mountPath: /run/fuse.ossfs
```

Comment:
connector currently using `/run/csi-tool/connector`, so maybe `/run/csi-tool/fuse.ossfs`? I want to ensure we will not conflict with the ossfs itself in the future.